docs/topics/api/accounts.rst

```diff
@@ -50,3 +50,57 @@ Returns information about your profile.
     :statuscode 200: profile found.
     :statuscode 400: an error occurred, check the `error` value in the JSON.
     :statuscode 401: authentication failed.
+
+--------------
+Super-creation
+--------------
+
+This allows you to generate a new user account and sign in as that user.
+
+.. important::
+
+    * Your API user must be in the ``Accounts:SuperCreate`` group to access
+      this endpoint.
+    * This endpoint is not available in all
+      :ref:`API environments <api-environments>`.
+
+.. http:post:: /api/v3/accounts/super-create/
+
+    **Request:**
+
+    :param email: assign the user a specific email address.
+        A fake email will be assigned by default.
+
+    .. sourcecode:: bash
+
+        curl https://addons.mozilla.org/api/v3/accounts/super-create/ \
+            -X POST -H 'Authorization: JWT <jwt-token>'
+
+    **Response:**
+
+    .. sourcecode:: json
+
+        {
+            "username": "super-created-7ee304ce",
+            "user_id": 10985,
+            "email": "super-created-7ee304ce@addons.mozilla.org",
+            "session_cookie": {
+                "encoded": "sessionid=.eJyrVopPLC3JiC8tTi2KT...",
+                "name": "sessionid",
+                "value": ".eJyrVopPLC3JiC8tTi2KT..."
+            }
+        }
+
+    :statuscode 201: Account created.
+    :statuscode 409: Incorrect request parameters.
```

Comment:
I like [`422`](https://httpstatuses.com/422) for this error. `409` would make sense if the email exists but not if it isn't a valid email. `422` works for both.